tools/releaseBuild/signing.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SignConfigXML>
+    <!-- ****Begin**** Authenticode - Authenticode SHA256 ******** -->
+    <job platform="" configuration="" dest="__OUTPATHROOT__\signed" jobname="PowerShell Core" approvers="vigarg;gstolt">
+        <file src="__INPATHROOT__\PowerShell.Core.Instrumentation.dll" signType="Authenticode" dest="__OUTPATHROOT__\PowerShell.Core.Instrumentation.dll" />
```

Comment:
you don't need the update signing step in your build with this xml... try disabling it.

Reply:
Tried. Does not work.

Reply:
thanks for trying